routes/v0/cases.js

```diff
@@ -14,7 +14,9 @@ casesRouter.get('/', async (req, res) => {
 });
 
 casesRouter.post('/', async (req, res) => {
-  const createdCase = await CaseController.createCase(req.body);
+  const { accountSid } = req;
```

Comment:
Here we can access this because `addAccountSid` middleware added this property to the request object. Same will happen to every request made against our API.